source/_patterns/02-molecules/jumbotron/_jumbotron.twig

```diff
@@ -26,17 +26,17 @@
     <div class="container">
   {% endif %}
 
-  {% block branding %}
+  {% block header_branding %}
```

Comment:
I'd very much like to name the jumbotron regions something quite different from the drupal regions. Call these `jumbotron_branding` and `jumbotron_cta`. This will make a clearer distinction between what is the design system and what is drupal stuff.